src/gui/tools/gt_qmltoolbargroup.h

```diff
@@ -0,0 +1,68 @@
+/* GTlab - Gas Turbine laboratory
+ *
+ * SPDX-License-Identifier: MPL-2.0+
+ * SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
+ *
+ * Author: Martin Siggel (AT-TWK)
+ */
+
+#ifndef GTQMLTOOLBARGROUP_H
+#define GTQMLTOOLBARGROUP_H
+
+#include <gt_qmlaction.h>
+
+#include <gt_listmodel.h>
+
+#include <gt_gui_exports.h>
+
+using ActionModel = GtListModel<GtQmlAction*>;
```

Comment:
Maybe we should use a more descriptive typename
```suggestion
using GtQmlActionModel = GtListModel<GtQmlAction*>;
```